### PR TITLE
🐛 Fixed decleration types does not support aliases

### LINKED ISSUE #1229: Missing intellisense values on placement property on typescript components
**Describe the bug**

**Steps to reproduce the bug**

1.
2.
3.
4.

**Expected behavior**

**Specifications**

- Version: 
- Browser: 
- OS: 

**Additional context**

Add any other context or screenshots about the bug.

### PR DESCRIPTION
**Pull request by mimarz**
fixes #1229

### CODE REVIEW

File: libraries/core-react/src/hooks/usePopper.ts [hunk 1]
```diff
@@ -1,25 +1,22 @@
 import type { CSSProperties } from 'react'
 import { usePopper as reactPopper } from 'react-popper'
 
-export const PlacementValues = [
```
Comment (pomfrida) [inline libraries/core-react/src/hooks/usePopper.ts:4]: If the props table in Popover still works without this, you can remove the argTypes from `Popover.stories`:

```
argTypes: {
    placement: {
      control: {
        type: 'select',
        options: PlacementValues,
        defaultValue: 'auto',
      },
    },
  },
```

Comment (mimarz) [reply libraries/core-react/src/hooks/usePopper.ts:4]: hmmm, I forgot to test the stories. 🙈 Let me check it out!

Comment (mimarz) [reply libraries/core-react/src/hooks/usePopper.ts:4]: Removed the `argsTypes` as it works without it now @pomfrida  :)

**Review by pomfrida — APPROVED** [review]
LGTM 👍

**Pull request merged**